app/subcommands/repo/list

```diff
@@ -41,7 +43,13 @@ for dir in "$DAB_CONF_PATH"/repo/*; do
 
 	url="$(dab config get repo/"$repo"/website)"
 
-	repo_row "$repo" "$status" "${clean:-}" "${uptodate:-}" "$url"
+	if [ "$filter" = "all" ]; then
```

Comment:
The contents of each statement is the same and the value of filter cannot trigger both blocks. Instead you could combine them so it is `if [ "$filter" = 'all' ] || [ "$filter" = "$status" ]; then`